### PR TITLE
Issue 12: Add package website

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR starts work on closing #12.

So far I have added a basic version of the `roxygen2` function families.

## Checklist

- [x] My PR is based on a package issue and I have explicitly linked it.
- [x] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [x] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [x] I have tested my changes locally.
- [x] I have added or updated unit tests where necessary.
- [x] I have updated the documentation if required.
- [x] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

**Comment by athowes** [comment]
@seabbs maybe you could help me out here as I missed the advanced GitHub training. Now that #18 is merged, I think there will have been changes to files that I also changed by adding the families in https://github.com/epinowcast/epidist/pull/19/commits/3079c84dc679ce04236bda94b6d111fd2efa8189. What's best practise here on whether I should pull in the changes or continue developing here and then they are pulled in once I am done? Perhaps I'm not expressing this clearly but I'd guess you know what I mean.

**Comment by seabbs** [comment]
Yes. So there are a few options. As we know your changes are unrelated I would probably keep going and we can resolve things out later. In general you would want to rebase this branch/pr from main prior to merging.

**Comment by athowes** [comment]
I've made the changes required I think to get the website started, and followed the git rebase and then force push workflow suggested. I think this should be OK to merge, and then after that we can get the website deployed. There is also work to do on the documentation etc. but that should be for separate issues / PRs.

File: _pkgdown.yml [hunk 1]
```diff
@@ -1,18 +1,39 @@
 url: epidist.epinowcast.org/
 template:
-  bootstrap: 5
-  bootswatch: flatly
-  theme: monokai
+  package: enwtheme
   twitter:
     creator: "@seabbs"
```
Comment (seabbs) [inline _pkgdown.yml:5]: This is boiler plate from Epinowcast that we probably don't need here

File: _pkgdown.yml [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+  contents:
+  - has_concept("model")
+- title: Postprocess
+  desc: Functions for postprocessing model output
+  contents:
+  - has_concept("postprocess")
+- title: Plot
+  desc: Functions and helper functions for plotting
+  contents:
+  - has_concept("plot")
```
Comment (seabbs) [inline _pkgdown.yml:39]: missing a new line here

File: _pkgdown.yml (hunk 1 above)
Comment (athowes) [reply _pkgdown.yml:5]: I don't fully understand the interface here as to what (which lines) this comment is pointing at?

**Pull request merged**